packages/libs/hd-wallet/src/SLIP10/kadenaGenKeypairFromSeed.ts

```diff
@@ -48,17 +56,17 @@ export function kadenaGenKeypairFromSeed(
  * @returns {([string, string] | [string, string][])} - Depending on the input, either a tuple for a single key pair or an array of tuples for a range of key pairs, with the private key encrypted if a password is provided.
  * @throws {Error} Throws an error if the seed buffer is not provided, if the indices are invalid, or if encryption fails.
  */
-export function kadenaGenKeypairFromSeed(
+export async function kadenaGenKeypairFromSeed(
   password: BinaryLike,
   seed: EncryptedString,
   indexOrRange: number | [number, number],
   derivationPathTemplate: string = `m'/44'/626'/<index>'`,
-): [string, EncryptedString] | Array<[string, EncryptedString]> {
+): Promise<[string, EncryptedString] | Array<[string, EncryptedString]>> {
   if (typeof seed !== 'string' || seed === '') {
     throw new Error('No seed provided.');
   }
 
-  const seedBuffer = kadenaDecrypt(password, seed);
+  const seedBuffer = new Uint8Array(await kadenaDecrypt(password, seed));
```

Comment:
doesn't kadenaDecrypt already return a Uint8Array?

Reply:
Yes I will fix this